run_e2e.sh

```diff
@@ -42,13 +42,16 @@ ginkgo_args=(
     "--noColor=true"
 )
 
+# Number of nodes in the cluster.
+NUM_NODES="${NUM_NODES:-$(kubectl get nodes --no-headers | wc -l)}"
```

Comment:
this will count control-plane nodes with `effect:NoSchedule` too.

Reply:
Thank you @neolit123! I realized why I thought `kubectl get nodes` returned only schedulable nodes in our Slack conversation -- sk8 doesn't deploy kubelets to control-plane only nodes. Those nodes run the API server, Controller Manager, and Scheduler as SystemD services without any kubelet involved.

It's may be accurate to then state that `kubectl get nodes` returns all registered kubelets.